### PR TITLE
fix: capture v2 (VF-2456)

### PR DESCRIPTION
**Pull request by DecathectZero**
**Fixes or implements VF-2456**

### Related PRs

| branch              | PR          |
| ------------------- | ----------- |
| other_pr_production | [link](url) |
| other_pr_master     | [link](url) |

### Checklist

- [ ] this is a breaking change and should publish a new major version
- [ ] appropriate tests have been written

### CODE REVIEW

File: packages/base-types/src/node/capture.ts [hunk 1]
```diff
@@ -17,7 +17,6 @@ export interface Step<Data = StepData> extends BaseStep<Data> {
 export interface Node extends BaseNode, NodeNextID {
   type: NodeType.CAPTURE;
   intent?: string;
-  slots?: string[];
```
Comment (DecathectZero) [inline packages/base-types/src/node/capture.ts:20]: this was just for an experiment I was running on the old capture steps.

File: packages/alexa-types/src/node/capture.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@ import { Voice } from '@/constants';
 
 export interface StepData extends Node.Capture.StepData<Voice> {}
```
Comment (z4o4z) [inline packages/alexa-types/src/node/capture.ts:7]: should we depreciate this one too?

File: packages/base-types/src/node/capture.ts [hunk 3]
```diff
@@ -10,14 +10,15 @@ export interface StepData {
   slotInputs: string[];
 }
```
Comment (z4o4z) [inline packages/base-types/src/node/capture.ts:12]: should we deprecate `StepData` too?

File: packages/base-types/src/node/captureV2.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  type: NodeType.CAPTURE_V2;
+}
+
+export interface Node extends BaseNode, NodeNextID {
+  type: NodeType.CAPTURE_V2;
+  intent?: {
+    name: string;
+    entities?: string[];
+  };
+  variable: string;
```
Comment (z4o4z) [inline packages/base-types/src/node/captureV2.ts:40]: I guess the variable should be optional
```suggestion
  variable?: string;
```

**Review by z4o4z — APPROVED** [review]

**Pull request merged**